components/dotenv.rst

```diff
@@ -62,9 +62,54 @@ The ``load()`` method never overwrites existing environment variables. Use the
     // ...
     $dotenv->overload(__DIR__.'/.env');
 
+As you're working with the Dotenv component you'll notice that you might want
+to have different files depending on the environment you're working in. Typically
+this happens for local development or Continuous Integration where you might
+want to have different files for your ``test`` and ``dev`` environments.
+
+You can use ``Dotenv::loadEnv()`` to ease this process::
+
+    use Symfony\Component\Dotenv\Dotenv;
+
+    $dotenv = new Dotenv();
+    $dotenv->loadEnv(__DIR__.'/.env');
+
+The Dotenv component will then look for the correct ``.env`` file to load
+in the following order whereas the files loaded later override the variables
```

Comment:
Are you sure about the overriding? Just by looking at the code I would assume the opposite (see https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Dotenv/Dotenv.php#L70 and https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Dotenv/Dotenv.php#L50).

Reply:
Yeah, they are overriding the other local variables. I just double checked with @nicolas-grekas and the `$overrideExistingVars` actually means overriding real environment variables. What we're talking about here are the variables defined by previously loaded `.env` files.